tilejson/tilejson.json.erb

```diff
@@ -17,7 +17,7 @@
     "name" : "<%= @params[:name] %>",
     "private" : true,
     "scheme" : "xyz",
-    "tilejson" : "2.0.0",
+    "tilejson" : "<%= @params[:tilejson_version] %>",
     "tiles" : [
         "http://<%= node[:tilejson][:host] %>/mapzen/vector/<%= node[:tilejson][:vector_tiles_version] %>/all/{z}/{x}/{y}.<%= @params[:extension] %>?api_key={api_key}"
```

Comment:
If this is related to mapzen/chef-mapzen_tilequeue#27 then does `:vector_tiles_version` need to change to `:url_path_version`?